plugins/auth-backend/src/service/router.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import express from 'express';
+import Router from 'express-promise-router';
+import { Logger } from 'winston';
+
+export interface RouterOptions {
+  logger: Logger;
+}
+
+export async function createRouter(
+  options: RouterOptions,
+): Promise<express.Router> {
+  const logger = options.logger.child({ plugin: 'auth' });
+  const router = Router();
+
+  router.get('/ping', async (req, res) => {
+    res.status(200).send('pong');
+  });
+
+  const app = express();
+  app.set('logger', logger);
+  app.use('/', router);
```

Comment:
Maybe make this `/auth` fairly early?

Reply:
since `/auth` is setup [here](https://github.com/spotify/backstage/pull/879/files/322d2939e5b0f4495c15ef90d56f3c5493711a23#diff-503528c3c492571833ada6e675d5ae29R67) 

I assumed this handles all the subroutes from /auth onwards.

cc @freben @Rugvip

Reply:
Yep, just `app.use(router)` though

Reply:
That's functionally the same isn't it